File: tests/python/simple.py
import os

class Color:
    RED = 0
    GREEN = 1
    BLUE = 2

def factorial(n):
    if n == 0:
        return 1
    else:
        return n * factorial(n - 1)

msg = "hello"
x = 42
b = True
print(factorial(x))

File: tests/rust/simple.rs
use std::fmt;

enum Color {
    Red,
    Green,
    Blue,
}

fn factorial(n: u64) -> u64 {
    if n == 0 {
        return 1;
    }
    n * factorial(n - 1)
}

fn main() {
    let msg = "hello";
    let x = 42;
    let b = true;
    println!("{}", factorial(x));
}
